tools/staramr/staramr_search.xml

```diff
@@ -89,11 +95,20 @@
                 </when>
                 <when value="none" />
             </conditional>
+            <conditional name="plasmidfinder_type">
+                <param name="plasmidfinder_type_condition" type="select" label="Specify PlasmidFinder Database type">
+                    <option value="default" selected="true">Include all available database types</option>
+                    <option value="gram_positive">Gram Positive</option>
+                    <option value="enterobacteriaceae">Enterobacteriaceae</option>
+                </param>
+            </conditional>
         </section>
     </inputs>
     <outputs>
         <data format="tabular" name="summary" label="${tool.name} on ${on_string}: summary.tsv" />
+        <data format="tabular" name="detailed_summary" label="${tool.name} on ${on_string}: detailed_summary.tsv" />
         <data format="tabular" name="resfinder" label="${tool.name} on ${on_string}: resfinder.tsv" />
+        <data format="tabular" name="plasmidfinder" label="${tool.name} on ${on_string}: plasmidfinder.tsv" />
         <data format="tabular" name="pointfinder" label="${tool.name} on ${on_string}: pointfinder.tsv">
             <filter>use_pointfinder['enable']</filter>
```

Comment:
You'll have to update this line to disable the display of the pointfinder output in case a user selects the option to not run pointfinder:

![image](https://user-images.githubusercontent.com/200517/56931325-3a31c880-6aa5-11e9-9fb7-786719382b77.png)


You can find more information about this type of parameter at https://planemo.readthedocs.io/en/latest/writing_advanced.html#static-outputs-determinable-from-inputs.

Reply:
I filtered it so that if the pointfinder is disabled it wouldn't include the pointfinder tsv file